xgi/dynamics/synchronization.py

```diff
@@ -63,10 +66,19 @@ def compute_kuramoto_order_parameter(H, k2, k3, w, theta, timesteps=10000, dt=0.
     triangles = H_int.edges.filterby("size", 3).members()
     n = H_int.num_nodes
 
-    r_time = np.zeros(timesteps)
+    theta_time = np.zeros((n, timesteps))
+    times = np.arange(timesteps) * dt
+
+    if omega is None:
+        omega = np.random.normal(0, 1, n)
+
+    if theta is None:
+        theta = np.random.normal(0, 1, n)
```

Comment:
Why not a random uniform on [0, 2pi]?

Reply:
Good point. Alexis used this in his PR so I added here too for consistency. Also thought about the uniform one, but perfectly uniform might be "dangerous" because with identical frequencies it is sometimes a stable state, so you won't see any convergence. Could do a uniform on [0, 2pi] + some small noise. Or leave it like this and let the user pick. What do you think?

Reply:
Updated to what you said, more natural imo